File: Java/1-50/10.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        long A, B, C, N, d;
        N = scanner.nextInt();
        A = scanner.nextInt();
        B = scanner.nextInt();
        C = scanner.nextInt();
        d = A + B + C;
        if (N <= d) {
            System.out.println("Yes");
        } else {
            System.out.println("No");
        }
    }
}

File: Kotlin/1-50/10.kt
import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)
    var A: Long
    var B: Long
    var C: Long
    var N: Long
    var d: Long
    N = scanner.nextLong()
    A = scanner.nextLong()
    B = scanner.nextLong()
    C = scanner.nextLong()
    d = A + B + C
    if (N <= d) {
        println("Yes")
    } else {
        println("No")
    }
}
